src/panels/developer-tools/event/event-subscribe-card.ts

```diff
@@ -74,7 +76,12 @@ class EventSubscribeCard extends LitElement {
                   ev.id
                 )}
                 ${formatTime(new Date(ev.event.time_fired), this.hass!.locale)}:
-                <pre>${JSON.stringify(ev.event, null, 4)}</pre>
+                <ha-code-editor
```

Comment:
use `ha-yaml-editor` instead with `.defaultValue=${ev.event}`, then you don't have to use `dump` as that is handled by the component and it is more efficient.